src/api/BrowserTypes.ts

```diff
@@ -5861,3 +5861,738 @@ const typeMap: any = {
         "WCP6Goodbye",
     ],
 };
+
+export function isAddContextListenerRequest(value: any): value is AddContextListenerRequest {
+    try {
+        Convert.addContextListenerRequestToJson(value);
```

Comment:
Are we happy with this "brute force" check of the types. It would be much harder to split the `Convert` class up to allow us to just check the conformance of rhte value rather than just trying to convert it to a string (which does conformance checking anyway).